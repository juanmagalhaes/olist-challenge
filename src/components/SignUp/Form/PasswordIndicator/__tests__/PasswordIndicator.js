import React from 'react';
import {
  PasswordIndicator,
} from '../PasswordIndicator';


test('renders', () => {
  expect(shallow(
    <PasswordIndicator
      isEmpty={true}
      passwordStrength={{}}
    />
  )).toMatchSnapshot();
});

