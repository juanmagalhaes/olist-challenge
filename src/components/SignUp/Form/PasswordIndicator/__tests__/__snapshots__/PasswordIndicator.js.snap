// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <Indicators />
  <BulletPoints
    isEmpty={true}
    passwordStrength={Object {}}
  />
</div>
`;
